metricbeat/docs/modules/beat.asciidoc

```diff
@@ -20,6 +20,9 @@ UI in {kib}. To enable this usage, set `xpack.enabled: true` and remove any `met
 from the module's configuration. Alternatively, run `metricbeat modules disable beat` and
 `metricbeat modules enable beat-xpack`.
 
+NOTE: This module sends metrics to the monitoring index instead of the default
```

Comment:
I understand from looking at the [preview](https://beats_23916.docs-preview.app.elstc.co/guide/en/beats/metricbeat/master/metricbeat-module-beat.html#_usage_for_stack_monitoring) that this note renders under the "Usage for Stack Monitoring" section, which talks about the `xpack.enabled: true` flag. For extra clarity, however, could we qualify this note by saying something like, "When this module is used for Stack Monitoring, it sends metrics...".